pkgs/development/tools/omnisharp-roslyn/default.nix

```
@@ -35,14 +35,14 @@ buildDotnetModule rec {
     "-property:AssemblyVersion=${version}.0"
     "-property:FileVersion=${version}.0"
     "-property:InformationalVersion=${version}"
-    "-property:RuntimeFrameworkVersion=6.0.0-preview.7.21317.1"
+    "-property:RuntimeFrameworkVersion=6.0.8"
```

If we're gonna replace what's in the omnisharp-roslyn build here we can just interpolate the sdk version here:
```suggestion
    "-property:RuntimeFrameworkVersion=${dotnet-sdk.version}"
```

---

This is the runtime version.  I expect it will have to be exposed by the sdk like `dotnet-sdk.runtimeVersion` or `dotnet-sdk.runtime.version`.

I wonder if the actual version of the `dotnet-runtime` package should be 6.0.8, or just match the sdk?  I'll have to investigate where that version comes from.

---

The version for the `dotnet-runtime` package should indeed be 6.0.8 as other things in nix might depend on it (and it is our current behavior for it)